example/src/Utilities/utils.ts

```diff
@@ -0,0 +1,5 @@
+export function isImageUrl(url: string): boolean {
+  const link = url.toLowerCase();
+  const imageExtension = ['jpg', 'jpeg', 'png'];
```

Comment:
how about if the URL includes "png" in the middle?
You can refer to this solution:
`
function isImageUrl(url) {
  const pattern = /(http(s?):)([/|.|\w|\s|-])*\.(?:jpg|gif|png|jpeg|bmp|webp|svg)$/;
  return pattern.test(url);
}
`